apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/editor-template-classes/index.js

```diff
@@ -31,16 +31,17 @@ const EditorTemplateClasses = withSelect( select => {
 	return { templateClasses };
 } )( ( { templateClasses } ) => {
 	const blockListInception = setInterval( () => {
-		const blockList = document.querySelector(
-			'.block-editor-writing-flow.editor-writing-flow > div'
-		);
+		const blockListParent = document.querySelector( '.block-editor__typewriter > div' );
```

Comment:
I notice we had to change the selector, but also 👍  to making the variable name a bit more semantic as well.

Reply:
thanks :)